tiny_dnn/layers/layer.h

```diff
@@ -310,6 +313,9 @@ class layer : public node {
    *important
    **/
   virtual serial_size_t fan_out_size() const { return out_shape()[0].width_; }
+  virtual serial_size_t fan_out_size(serial_size_t) const {
+    return fan_out_size();
+  }
```

Comment:
If you could write sigle line comments above new layer methods, that should be helpful.